app/jobs/unzip_job.rb

```diff
@@ -36,11 +36,7 @@ def perform(work_version)
 
   # determines if we should actually unzip this file, based on it not being in globus and being a zip file
   def unzip_file?(zip_attached_file)
-    !zip_attached_file.in_globus? && zip_mime_types.include?(zip_attached_file.content_type)
-  end
-
-  def zip_mime_types
-    ['application/zip', 'application/x-zip-compressed', 'application/x-zip', 'multipart/x-zip']
+    !zip_attached_file.in_globus? && zip_attached_file.zip?
   end
```

Comment:
move this logic from the job to the attached file class so we can have access to it in our new validation